kevm_pyk/src/kevm_pyk/__main__.py

```diff
@@ -65,12 +65,18 @@ def main():
             if args.command == 'solc-to-k':
```

Comment:
This function needs a clean up, please create an issue for that.

Reply:
Recorded: https://github.com/runtimeverification/k/issues/2801